public/src/canadian-championship-bidding-policy.md

```diff
@@ -60,6 +62,13 @@ Applications that are being submitted on behalf of a larger organization team **
   - The review process _may_ include virtual meetings among the Canadian Championship Selection Committee.
   - Virtual meetings between the Canadian Championship Selection Committee and each organization team _may_ be proposed before voting has begun.
     - Modifications _may_ be made to an application if it is required by the Canadian Championship Selection Committee.
+  - The voting process **must** be conducted using a ranked ballot system.
+    - If a member of the Canadian Championship Selection Committe is part of the Core Team for an Application, they are **not** eligible to vote.
+    - Each member of the Canadian Championship Selection Committee will receive a ballot listing all submitted applications.
+    - Committee members *must* rank the applications in order of preference, assigning a unique rank to each application.
+      - 1 will be considered the highest preference, with each descending number representing a lower preference.
+    - Once all committee members have submitted their ballots, the votes will be tabulated. One application *must* receive a majority of votes to win.
+      - If a vote leads to a tie, the Leader of the Canadian Championship Selection Committee will select the host of the Canadian Championship.
```

Comment:
I thought a ranked ballot system worked by continually eliminating the group(s) with the fewest votes, and redistributing those peoples' votes to their next (non-eliminated) choice, until either a group has a majority or all groups remaining have the same number of votes.

But the old PR had
> In the case of a time, the bid with the highest number of second-choice rankings will be declared the winner. If a tie persists, the bid with the highest number of third-choice rankings will be declared the winner. This process will continue until the tie is broken.

(which was removed before merging, and seems bad because the 2 best venues could split first-choice and second-choice rankings and then a 3rd venue could get all the third-choice rankings, making it winner).

If there's no universal definition of "ranked ballot system" then we should specify what we mean.